packages/react-app/src/constants.js

```diff
@@ -71,9 +71,9 @@ export const NETWORKS = {
     chainId: 10200,
     price: 1,
     gasPrice: 1000000000,
-    rpcUrl: "https://rpc.chiadochain.net",
-    faucet: "https://gnosisfaucet.com",
-    blockExplorer: "https://blockscout.chiadochain.net",
+    rpcUrl: "https://rpc.chiadochain.net/",
```

Comment:
![Screenshot 2023-01-27 at 4 31 09 PM](https://user-images.githubusercontent.com/9419140/215204428-623862b3-1aa2-4700-a712-c0dff5421d1d.png)

@skaunov I noticed the trailing slash was only on the gnosis mainnet 🤔

Reply:
Yeah, but our own Scaffold components pretty much enjoy failing in quite unexpected (to me) places when links are without trailing slash. It's from bitter experience. =(